engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/CreditsScreen.java

```diff
@@ -0,0 +1,63 @@
giomieic' Domingues\n MaloJaffre\n AlexanderGrooff\n janzegers\n Bas 'BdeBock' de Böck\n Arkka 'arkka' Dhiratara\n Rostyslav 'rzats' Zatserkovnyi\n Taha Doğan 'tdgunes' Güneş\n Omer 'oijazsh' Sheikh\n Mehul 'HashCode55' Ahuja\n JeanCGF\n David 'dkambersky' Kamberský\n Mihalache 'Margretor' Octavian\n oniatus\n Skylar 'SkySom' Sommers\n Abhinav 'genuinelucifer' Tripathi\n TobyShaw\n ClintRajaniemi\n Leonardo Akira 'Leonardoas26' Shimabukuro\n Michael 'pollend' Pollind\n Arsenii 'kujiraOo' Kurilov\n Adam 'planetguy32' Price\n portokaliu / Anthodeus\n Jessica 'LadySerenaKitty' Hawkwell\n Milan 'CptCrispyCrunchy' Ender\n Martin 'Vessalix' van Kuik\n Viveret 'viveret' Amant\n tyuiwei / korp\n Ian 'Motta' Maltbey\n joseph-healy\n engiValk\n XTariq\n roshikouhai\n Arcnor\n naiffuR\n Axydlbaaxr\n Waterpicker\n indianajohn\n Kazimir3701\n Translator5\n XroMyla\n Koga Masato\n kaen\n Avalancs\n jchappelle\n qwc\n coty91 / NullThought\n kartikey0303\n nihal111\n segfault802\n rodrigorosa\n triebben\n arpan98\n nclsppr\n namanyadav12\n zeokav\n nahh\n ron-popov\n MaxBorsch\n freyley\n Tropid\n HolixSF\n agsmith\n Stefan-Mitic\n cribsb\n \n GUI Team\n \n  Anton \"small-jeeper\" Kireev\n miniME89\n Lucas 'x3ro' Jenß\n Piotr 'Halamix2' Halama\n \n Logistics Team\n \n  AlbireoX\n Mathias Kalb\n Richard \"rapodaca\" Apodaca\n Stellarfirefly\n Mathias 'mkalb' Kalb\n MrBarsack\n Dennis 'Philaxx' Urban\n 3000Lane\n MiJyn\n Pavel 'neoascetic' Puchkin\n \n World Team\n \n  bi0hax\n ddr2\n Nym Traveel\n Tobias 'Skaldarnar' Nett\n Tenson\n Laurimann\n MatthewPratt\n Martin 'msteiger' Steiger\n Josh 'Josharias' Zacharias\n \n Soundtrack and Sound Effects\n \n Primary soundtrack by Chris Köbke - https://soundcloud.com/chriskoebke\n Sunrise, Afternoon and Sunset composed by Karina Kireev\n Dimlight, Resurface and Other Side composed and produced by Exile.\n Door Open sound by Pagancow, from FreeSound.org\n Door Close sound by Fresco, from FreeSound.org\n Camera Click Noise from Snapper4298, from FreeSound.org\n Other sound effects created by Exile.\n \n Icons from \"Fugue Icons\" by Yusuke Kamiyamane (CC BY 3.0)\n \n editor_array.png\n editor_attribute.png\n editor_object.png\n editor_uibox.png\n editor_uibutton.png\n editor_uicheckbox.png\n editor_uidoubleslider.png\n editor_uidropdown.png\n editor_uidropdownscrollable.png\n editor_uiimage.png\n editor_uilabel.png\n editor_uilist.png\n editor_uiloadbar.png\n editor_uiscrollbar.png\n editor_uislider.png\n editor_uitext.png\n editor_uitextentry.png\n editor_uitooltip.png\n editor_uitreeview.png\n editor_zoomablelayout.png\n \n Icons by Florian Köberle (CC BY 4.0)\n \n editor_cardLayout.png\n editor_columnlayout.png\n editor_flowlayout.png\n editor_miglayout.png\n editor_relativelayout.png\n editor_rowlayout.png\n editor_uispace.png\n contract.png\n contractOver.png\n expand.png\n expandOver.png\n \n Icons prepared by kartikey0303 (CC BY 4.0)\n \n checkboxChecked.png\n checkboxCheckedDisabled.png\n checkboxCheckedHover.png";
```

Comment:
Yep, this needs to be moved somewhere. I think a `Credits.txt` / `Credits.html` / `Credits.md` (if you implement support for the latter two) file in the root of the `/resources` directory would be appropriate, but this is up for debate.